src/index.css

```diff
@@ -3,6 +3,25 @@ body {
   padding: 0;
 }
 
+a[href^="http"]::after,
+a[href^="https://"]::after
+{
+  content: "";
+  width: var(--goa-icon-size-l);
+  height: var(--goa-icon-size-l);
+  color: var(--goa-color-interactive-default);
+  background: url("data:image/svg+xml,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20class%3D%22ionicon%22%20viewBox%3D%220%200%20512%20512%22%3E%0A%09%3Cpath%20%0A%09%09d%3D%22M384%20224v184a40%2040%200%2001-40%2040H104a40%2040%200%2001-40-40V168a40%2040%200%200140-40h167.48M336%2064h112v112M224%20288L440%2072%22%20%0A%09%09fill%3D%22none%22%20%0A%09%09stroke%3D%22%230070c4%22%20%0A%09%09stroke-linecap%3D%22round%22%20%0A%09%09stroke-linejoin%3D%22round%22%20%0A%09%09stroke-width%3D%2232%22%0A%09%2F%3E%0A%3C%2Fsvg%3E%0A")
+    center bottom no-repeat;
+  display: inline-block;
+  margin-left: var(--goa-space-xs);
+  vertical-align: middle;
+}
+
+a[href^="https://"]:hover:after {
```

Comment:
Could you merge this background image with the other then set the non-hover styles in a separate style block?

Reply:
I believe if the stroke color is set to `currentColor` (within the svg) you can override it with the css styles of `stroke` or `color`

Reply:
```
<svg xmlns="http://www.w3.org/2000/svg" class="ionicon" viewBox="0 0 512 512">
  <path td="M384 224v184a40 40 0 01-40 40H104a40 40 0 01-40-40V168a40 40 0 0140-40h167.48M336 64h112v112M224 288L440 72" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="32"/>
</svg>
```